diff --git a/client3.py b/client3.py
index 818d23f..de8d053 100644
--- a/client3.py
+++ b/client3.py
@@ -42,7 +42,9 @@ def getDataPoint(quote):
 def getRatio(price_a, price_b):
     """ Get ratio of price_a and price_b """
     """ ------------- Update this function ------------- """
-    return 1
+    if(price_b==0):
+        return
+    return price_a/price_b
 
 
 # Main
diff --git a/mypatch.patch b/mypatch.patch
index 196e05e..e69de29 100644
--- a/mypatch.patch
+++ b/mypatch.patch
@@ -1,13 +0,0 @@
diff --git a/client3.py b/client3.py
index 3fc09b7..818d23f 100644
--- a/client3.py
+++ b/client3.py
@@ -35,7 +35,7 @@ def getDataPoint(quote):
     stock = quote['stock']
     bid_price = float(quote['top_bid']['price'])
     ask_price = float(quote['top_ask']['price'])
-    price = bid_price
+    price = (bid_price+ask_price)/2
     return stock, bid_price, ask_price, price
